bootstrap/file/file.go

```diff
@@ -6,12 +6,13 @@ import (
 	"net/http"
 	"net/url"
 	"os"
-	"time"
 
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/environment"
 	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/clients/logger"
 )
 
-func Load(path string, timeout time.Duration, provider interfaces.SecretProvider) ([]byte, error) {
+func Load(path string, provider interfaces.SecretProvider, lc logger.LoggingClient) ([]byte, error) {
```

Comment:
do we need or want to change the order of the input parameters?

Reply:
nope